gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/E2EMappingIT.java

```diff
@@ -46,7 +46,6 @@
 public class E2EMappingIT extends BaseEhrHandler {
 
     private static final boolean OVERWRITE_EXPECTED_JSON = false;
-    private static final int NHS_NUMBER_MIN_MAX_LENGTH = 10;
```

Comment:
Should this of been removed?

Reply:
It already exists in the base class where it gets used ,so I removed it after getting sonar lint comments?